app/code/Magento/InventoryCatalogAdminUi/etc/config.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
should be moved to InventoryCatalog module, as we should support Headless MSI usage (without AdminUi modules installed)